.github/workflows/buildAndTest.yml

```diff
@@ -0,0 +1,78 @@
+name: Build & Test
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  scalafmt:
+    name: Check formatting
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cache SBT ivy cache
+        uses: actions/cache@v1
+        with:
+          path: ~/.ivy2/cache
+          key: ${{ runner.os }}-sbt-ivy-cache-${{ hashFiles('**/build.sbt') }}
+      - name: Cache SBT
+        uses: actions/cache@v1
+        with:
+          path: ~/.sbt
+          key: ${{ runner.os }}-sbt-${{ hashFiles('**/build.sbt') }}
+      - name: Scalafmt
+        run: sbt scalafmtCheckAll
+
+  test:
+    name: Compile and Test
```

Comment:
I would split `compile` and `test`. I know that `test` also compiles, but the trade-off is: The machine works more, but the human has to work less, because she know directly which of the two failed.

Reply:
Done in https://github.com/moia-dev/scala-http-client/pull/52/commits/02eeeaa1f54ce082bbec1633aa8d32062f1c1612.